source/installation/resource-manager/kubernetes.rst

```diff
@@ -91,12 +91,66 @@ cluster configuration above or specify them for each app individually.
 
 You should have policies in place to enforce these.
 
-Bootstrapping an Open OnDemand Kuberenetes cluster
-**************************************************
+Bootstrapping the Kuberenetes cluster
+*************************************
 
 Before anyone can use your Kubernetes cluster from Open OnDemand, you'll need
 to create the `open ondemand kubernetes resources`_ on your cluster.
 
+Below is an example of adding the necessary resources:
+
+.. code-block:: sh
+
+  kubectl apply -f https://raw.githubusercontent.com/OSC/ondemand/master/hooks/k8s-bootstrap/ondemand.yaml
+
+
+Bootstrapping OnDemand web node to communicate with Kubernetes
+**************************************************************
+
+The OnDemand web node ``root`` user must be configured
+to use the ``ondemand`` service account deployed by the `open ondemand kubernetes resources`_ and
+be able to execute ``kubectl`` commands.
+
+First deploy ``kubectl`` to the OnDemand web node:
+
+.. code-block:: sh
+
+  wget -O /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.21.5/bin/linux/amd64/kubectl
```

Comment:
They should be version aware here.

Reply:
Do you mean like replace version with like `$KUBERNETES_VERSION` or something similar?

Reply:
Just a note that this URL has a version in it.  If you think folks would be able to spot that, then we can keep it as is.

Reply:
I think I got a commit to do this.

Reply:
I made it clear that `$VERSION` should be replaced by version of controller.